examples/seismic/tutorials/11_viscoacoustic.ipynb

```diff
@@ -0,0 +1,907 @@
+      "text/plain": [
+       "<Figure size 576x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "# Create model\n",
+    "model = ModelViscoacoustic(space_order=space_order, vp=v, qp=qp, rho=rho, \n",
+    "                           origin=origin, shape=shape, spacing=spacing, \n",
+    "                           nbl=nbl)\n",
+    "plot_velocity(model)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f0 = 0.010 # peak/dominant frequency \n",
+    "irho = model.irho\n",
+    "rho = 1./irho\n",
+    "\n",
+    "# velocity model\n",
+    "vp = model.vp\n",
+    "\n",
+    "t_s = (sp.sqrt(1.+1./model.qp**2)-1./model.qp)/f0\n",
+    "t_ep = 1./(f0**2*t_s)\n",
+    "tt = (t_ep/t_s) - 1.\n",
+    "\n",
+    "s = model.grid.stepping_dim.spacing\n",
+    "damp = model.damp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Time step in ms and time range:\n",
+    "t0, tn = 0., 4000.\n",
```

Comment:
I think 2000. (2sec) is better from running the notebook so that the wavefield snapshot lokks nice and really shows the difference between the three as currently only see boundary reflections.
To plot the wavefield, I wouldn't (personnally) used `plot_image` that isn't really intended for it, you can look at h`06_bis` to see how to plot wavefield in a nice way or just do a simple `imshow` but with `camp="seismic"`